ArduCopter/RC_Channel.cpp

```diff
@@ -252,8 +252,10 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
 #if RANGEFINDER_ENABLED == ENABLED
             if ((ch_flag == HIGH) && copter.rangefinder.has_orientation(ROTATION_PITCH_270)) {
                 copter.rangefinder_state.enabled = true;
+                copter.gcs().send_text(MAV_SEVERITY_WARNING, "SurfaceTracking enabled");
```

Comment:
```suggestion
                gcs().send_text(MAV_SEVERITY_WARNING, "SurfaceTracking enabled");
```

gcs() should be available from any context

Reply:
was just copying it from the surface tracking module I was also using at one point for messaging while doing rf work....did not know why it had to specify copter either, but it worked....will change...